crates/kona/registry/etc/configs.json

```
@@ -101,6 +203,83 @@
             "PreimageOracle": "0x3052f7e497870bE9aD7cd2C1fFC415c5Fb8ecFC3",
             "DataAvailabilityChallenge": null
           }
+        },
+        {
+          "Name": "Celo Alfajores",
+          "PublicRPC": "https://alfajores-forno.celo-testnet.org",
+          "SequencerRPC": "https://sequencer.alfajores.celo-testnet.org",
+          "Explorer": "https://celo-alfajores.blockscout.com",
+          "SuperchainLevel": 0,
+          "GovernedByOptimism": false,
+          "DataAvailabilityType": "alt-da",
+          "l2_chain_id": 44787,
+          "batch_inbox_address": "0xff00000000000000000000000000000000044787",
+          "block_time": 1,
+          "seq_window_size": 3600,
+          "max_sequencer_drift": 1800,
+          "GasPayingToken": "0xF194afDf50B03e69Bd7D057c1Aa9e10c9954E4C9",
```

This is currently referencing the L2 token, but it should be referencing the L1 custom gas token proxy [here](https://storage.googleapis.com/cel2-rollup-files/alfajores/deployment-l1.json) 
```suggestion
          "GasPayingToken": "0x3E3FEA3F31ff162a460f11Af4c53f39E743fd88c",
```